### PR TITLE
pkg(befunge93-0.2.0) added befunge93

### PR DESCRIPTION
**Pull request by Hydrazer**
added befunge93 with python2 interpreter should work i think

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
         "prettier": "2.4.1"
+    },
+    "dependencies": {
+        "axios": "^0.24.0",
+        "chalk": "^4.1.2",
+        "minimatch": "^3.0.4",
+        "nocamel": "^1.0.2",
+        "semver": "^7.3.5",
+        "ws": "^8.3.0",
+        "yargs": "^17.2.1"
```
Comment (HexF) [inline package.json:12]: Whats with this?

File: packages/befunge93/1.0.0/build.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+source ../../python/2.7.18/build.sh
```
Comment (HexF) [inline packages/befunge93/1.0.0/build.sh:3]: Can we use python3?

File: packages/befunge93/1.0.0/build.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+git clone -q 'https://github.com/programble/befungee' befunge93
```
Comment (HexF) [inline packages/befunge93/1.0.0/build.sh:5]: Checkout an exact tag, in this case latest is `v0.2.0`

File: packages/befunge93/1.0.0/metadata.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+    "language": "befunge93",
+    "version": "1.0.0",
```
Comment (HexF) [inline packages/befunge93/1.0.0/metadata.json:3]: Match with version in the py file - `__version__`

**Review by HexF — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Hydrazer) [reply package.json]: no idea lol it told me to add those but i removed them now

File: packages/befunge93/1.0.0/build.sh (hunk 2 above)
Comment (Hydrazer) [reply packages/befunge93/1.0.0/build.sh]: i don't think so because they used print without brackets `print "File too large"`

**Review by HexF — APPROVED** [review]

File: packages/befunge93/0.2.0/build.sh [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+source ../../python/2.7.18/build.sh
+
+git clone -q 'https://github.com/programble/befungee' befunge93
```
Comment (HexF) [inline packages/befunge93/0.2.0/build.sh:5]: Can we checkout the exact tag?

**Review by HexF — CHANGES_REQUESTED** [review]

File: packages/befunge93/0.2.0/build.sh (hunk 5 above)
Comment (Hydrazer) [reply packages/befunge93/0.2.0/build.sh:5]: wdym

Comment (HexF) [reply packages/befunge93/0.2.0/build.sh:5]: Right now you are cloning master, which for all we know could be `v1.0.1`, so you need to pin an exact tag with `git checkout tag/v0.2.0`

Comment (HexF) [reply packages/befunge93/0.2.0/build.sh:5]: that is, after you clone

**Pull request closed without merging**